### ISSUE TITLE
Suite P v7.8.3 Call Edit View Contact/Lead Search doesn't work

### ISSUE THREAD

**Issue by bunk3m** [salesagility/SuiteCRM#3399]
#### Issue
In most of the detail views, you can attach a contact or lead to the task, meeting, call etc.

In the Call Edit, when you add any part of a name in the name search fields (first/last/email), the invite button action is to leave the page.  When you leave the page you lose all the data from the call that you were trying to create.  In the meeting or task detail view, when you push the invite search button, you are presented with a list of potential contacts/leads.

#### Expected Behavior
In the Call detail view, you should be able to search for and add a contact or lead to a call using the Invite search button.

#### Actual Behavior
In the Call detail view, when you put any combination of first name/last name/email, you are taken to another page and lose the data in the call detail view that you have added since the last save.

#### Possible Fix
What appears to work is to add the contact or lead in the top part of the screen, save and then go back to edit the call detail and then change the drop down box to Account and add the account.  This appears to work.

#### Steps to Reproduce
1. Select theme "Suite P"
2. Login again to enable the theme
3. In the "Create" dropdown top right menu, select "Log Call" to go to the Call Edit view
4. Add some info in the top of the screen, go to the Add Invitee
5. Type in firstname/lastname/email and press the "Search" button to the right of the input boxes.
6. You will see what is shown in the attached screenshot.
<img width="1425" alt="screen shot 2017-04-13 at 13 36 50" src="https://cloud.githubusercontent.com/assets/3494567/25017608/6980f448-2052-11e7-8f77-7a57697d2b16.png">
7. If you cancel you can't add an invitee
8. If you leave the page, you lose all the data you entered and you don't get to a search results list.

#### Context
This wastes a lot of time when I do the workaround reducing my productivity.
This is broken functionality that used to work OK.  I suggest that it is high priority.

#### Your Environment
* SuiteCRM Version used: 7.8.3
* FF 52.02 (64 bit) & Chrome 57.0.2987.133 (64-bit)
* mysql  Ver 14.14 Distrib 5.5.54, for debian-linux-gnu (x86_64) using readline 6.2, PHP 5.4.45-0+deb7u8
* Debian 7

**Comment by samus-aran** [comment]
Hi @bunk3m 

We are unable to replicate this on the demo instance which is running on 7.8.3.  
Can you test there and confirm that it works correctly for you?

Also, I'm just changing the title and the 'Detail View' part of this issue as what you are viewing is actually the Edit View.

My only thinking is that you are currently running PHP 5.4 so we will need to confirm if it is any issue with that.

Is this a fresh instance or an upgrade?

**Comment by bunk3m** [comment]
Hi.
I tested the same steps on the demo and it does work correctly.
I tried the same steps again both using "Suite P" and "Suite R" and both have the same problem with Firefox 52.02.  I usually use Firefox.

Just for a test, I tried Chrome.  Chrome works OK.  No problem with Chrome.
The issue appears to be with Firefox.  

I have cache for Firefox set to 0 (zero) bytes so it shouldn't be caching anything.

The Suite 7.8.3 instance is an upgrade.  It has been upgraded regularly since Sugar 6.5.x

I don't typically do RepairJS after an upgrade but will try now.  
The repair didn't fix the issue with Firefox.

So it looks like this is a problem using Firefox.  I guess as a work around I'll use Chrome but would prefer to figure out why Firefox doesn't work right.
